src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -307,8 +310,8 @@ protected Boolean scrollToSelector(String selector) {
       JavascriptExecutor js = (JavascriptExecutor) driver;
       try {
         js.executeScript("var x = $(arguments[0]);"
-            + "window.scroll(0,x.position()['top']+x.height()+100);"
-            + "$(window).trigger('scroll');", selector);
+                         + "window.scroll(0,x.position()['top']+x.height()+100);"
+                         + "$(window).trigger('scroll');", selector);
       } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)